utils/comment.go

```diff
@@ -176,37 +207,43 @@ func DeleteExistingPullRequestReviewComments(repo *Repository, pullRequestID int
 
 func getFrogbotComments(writer outputwriter.OutputWriter, existingComments []vcsclient.CommentInfo) (reviewComments []vcsclient.CommentInfo) {
 	for _, comment := range existingComments {
-		if outputwriter.IsFrogbotComment(comment.Content) || outputwriter.IsFrogbotSummaryComment(writer, comment.Content) {
```

Comment:
Delete writer outputwriter.OutputWriter arg fro function's signature since it is no longer in use